micro_g/package.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+
+  <name>micro_g</name>
+  <version>0.0.1</version>
+  <description>Metapackage for micro-g</description>
+
+  <maintainer email="evanp922@gmail.com">Evan Palmer</maintainer>
+  <maintainer email="charles.dwsn@gmail.com">Charles Dawson</maintainer>
```

Comment:
@dawsonc Let me know if you want me to change your email address here

Reply:
lgtm